src/operator/nn/mkldnn/mkldnn_convolution.cc

```diff
@@ -113,16 +113,28 @@ std::shared_ptr<mkldnn::convolution_forward::primitive_desc> GetConvFwdImpl(
       [&param, &data, &weights, &output, &attr](const mkldnn::convolution_forward::desc& desc) {
         auto engine = CpuEngine::Get()->get_engine();
         try {
-          // MKL-DNN introduced padded formats since 0.15 which require more memory
-          // compared to the actual size of the tensor. Currently, MKL-DNN operators
-          // still reuse memory from memory planning, so here we need to select a
-          // suboptimal kernel for computation that has the expected memory size requirements
+          // MKLDNN introduced padded formats since 0.15 which require more
```

Comment:
please keep  max line with  equal 100 in autoformater

Reply:
Good catch. I forgot that there is no _.clang-format_ file in v1.x repository.